docs/dev/json.rst

```diff
@@ -175,3 +175,62 @@ submitted via the ``additional_info`` element. We're open to see how people
 will make use of these unspecified fields and may decide to formally specify or
 scrub elements in the future.
 
+Artist Statistics JSON
```

Comment:
I'm not sure it makes sense to put this in the json file which is pretty big already.  Do you think we could do a different file or put it in the endpoint documentation itself.